src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -55,4 +55,13 @@ public String getNextLine() {
     public void closeScanner() {
         this.scanner.close();
     }
+
+    public void viewAccount(String accDetails) {
+        String name = accDetails.split(";")[0];
+        String bal = accDetails.split(";")[1];
+        System.out.println("----------------------------");
```

Comment:
Extract out into a method in ui class called printLine or smthing. Dont repeat yourself, as they all say

Reply:
have made the changes and pushed again, Thanks!